doc/tutorials/objdetect/aruco_board_detection/aruco_board_detection.markdown

```diff
@@ -0,0 +1,298 @@
+Detection of ArUco Boards {#tutorial_aruco_board_detection}
+==============================
+
+@prev_tutorial{tutorial_aruco_detection}
+@next_tutorial{tutorial_charuco_detection}
+
+An ArUco Board is a set of markers that acts like a single marker in the sense that it provides a
+single pose for the camera.
+
+The most popular board is the one with all the markers in the same plane, since it can be easily printed:
+
+![](images/gboriginal.png)
```

Comment:
Real life photos must be in JPEG format.
PNG blows up size in these cases.

Reply:
fixed

Reply:
The same is true for intermediate images that show algorithm steps.

Reply:
The last remaining PNG images are smaller in size than JPG images.